maxim/logger/livekit/module_functions.py

```
@@ -0,0 +1,109 @@
+import functools
+import inspect
+from datetime import datetime, timezone
+
+from livekit.agents import AgentSession
+from livekit.agents.voice.generation import ToolContext, _ToolOutput
+
+from ...scribe import scribe
+from .store import get_session_store
+
+
+def intercept_execute_tools_task_result(
```

Ruff (0.11.9)

Missing return type annotation for public function `intercept_execute_tools_task_result`

Add return type annotation: `None`

(ANN201)